adagrams/game.py

```diff
@@ -1,11 +1,79 @@
+import random
+from collections import Counter
+LETTER_POOL20 = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
 def draw_letters():
-    pass
+    values = list(LETTER_POOL20.values())
+    keys = list(LETTER_POOL20.keys())
+    players_hand =random.sample(keys, counts = values, k =10)
+    return players_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
-
+    word = word.upper()
+    available_bank = Counter(letter_bank)
+    for letter in word:
+        if letter in letter_bank and available_bank[letter]:
+            available_bank[letter] -= 1
+            available = True
+        else:
+            return False
+    return available
+            
 def score_word(word):
-    pass
-
+    word = word.upper()
+    point_system = {
```

Comment:
Since this is a constant variable, you can rename this variable to express that by using caps POINT_SYSTEM. You can also pull this variable outside of score_word, if you'd like to make it a global constant so that this function doesn't look so long